ui/main/src/app/services/lightcards/lightcards-feed-filter.service.ts

```diff
@@ -63,6 +67,10 @@ export class LightCardsFeedFilterService {
         return this.filteredLightCardsForTimeLine.asObservable();
     }
 
+    public getFilteredAndSearchedLightCards() : Observable<any> {
+        return this.filteredAndSearchedLightCards.asObservable();
+    }
+
     private computeFilteredLightCards() {
```

Comment:
In ui/main/src/app/modules/feed/components/map/map.component.ts  line 65 getFilteredAndSortedLightCards shall be replace by   getFilteredAndSearchedLightCards

In ui/main/src/app/modules/feed/feed.component.ts line 68 getFilteredAndSortedLightCards shall be replace by getFilteredAndSearchedLightCards

So method public getFilteredAndSearchedLightCards() should not be neeeded anymore in this service

Reply:
If we do this than the sort function doesn't work anymore.

Reply:
yes you are right , forgot my remark